test/src/screens/home_screen_test.dart

```diff
@@ -1 +1,140 @@
-void main() {}
\ No newline at end of file
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:van_quang_tinh/src/app.dart';
+
+void main() {
+  const colorOfTextAndIndicator = Color(0xff8FC746);
+  const labelStyle = TextStyle(fontWeight: FontWeight.w500, fontSize: 18);
+  final _tabs = ['Cryptocurrency', 'Categories'];
+
+  var widget = const MaterialApp(
+    home: App(),
+  );
+
+  group('Home Screen Widget Test', () {
```

Comment:
because you are writing test on home_screen_test so you don't need to have this group

Reply:
i fixed it